db/migrate/20201001095312_create_navigation_links.rb

```diff
@@ -0,0 +1,12 @@
+class CreateNavigationLinks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :navigation_links do |t|
```

Comment:
```
validates :name, :url, :icon, presence: true
```
Can we add these validations at the database level as well?

Reply:
@mstruve I've updated the migration here to allow for validation and I've just combined the two migration files into one whilst nobody has really pulled down the branch as yet and altered their local db.  Does this look good?